src/cc_toolchain/BUILD

```diff
@@ -54,56 +54,57 @@ filegroup(
 )
 
 filegroup(
-    name = "clang_wrappers",
-    srcs = glob(["wrapper/clang-*"]),
+    name = "k8_gcc_wrappers",
+    srcs = glob(["wrapper/k8_gcc-*"]),
 )
 
 filegroup(
-    name = "clang_all",
+    name = "k8_gcc_all",
     srcs = [
-        ":clang_wrappers",
-        "@llvm_clang//:binaries",
-        "@llvm_clang//:clang_libs",
-        "@llvm_clang//:includes",
-        "@llvm_clang//:runtime_libs",
-        "@llvm_clang//:static_libs",
+        ":k8_gcc_wrappers",
+        "@k8_gcc//:binaries",
+        "@k8_gcc//:k8_gcc_libs",
+        "@k8_gcc//:includes",
+        "@k8_gcc//:runtime_libs",
+        "@k8_gcc//:static_libs",
     ],
 )
 
-cc_toolchain_config(
-    name = "clang-k8",
+cc_toolchain_config_k8(
+    name = "k8_gcc",
     builtin_include_directories = [
-        "external/llvm_clang/include/c++/v1/",
+        "/usr/lib/gcc/x86_64-linux-gnu/",
         "/usr/include/",
         "/usr/local/include/",
+        "/usr/lib/gcc/c++/7.4.0/include/",
```

Comment:
Just to be clear, we explicitly have the include directory for `7.4.0` because we still depend on libs / code built with that version of `gcc`, so this is where we need to look to find it? Do we have any guarantee this folder exists?

Reply:
I think ubuntu 18.04 has gcc 7.4.0 preinstalled

Reply:
Perhaps we should install gcc 7.4.0 explicitly in the setup scripts @garethellis0 ? I don't reall  want to have stuff just magically assumed.

Reply:
Fine with me, installing `gcc-7` should do the trick I think.

Reply:
ok done